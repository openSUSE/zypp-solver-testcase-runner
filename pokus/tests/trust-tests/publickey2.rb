#!/usr/bin/ruby


# Check exception when importing private key as public key.


require 'zypp'
include Zypp


begin
    path = Pathname.new("../../data/privkey.txt")
    publickey = PublicKey.new(path)
    puts publickey.to_s()
rescue ZYppException => e
    puts "ZYppException caught"
    puts e.to_s
else
    puts "Oh, no exception caught"
    exit 1
end

