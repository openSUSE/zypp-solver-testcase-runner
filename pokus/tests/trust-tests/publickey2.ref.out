ZYppException caught
File ./../../data/privkey.txt doesn't contain public key data
